sparkle/tools/solver_wrapper_parsing.py

```diff
@@ -63,6 +63,9 @@ def get_solver_call_params(args_dict: dict, prefix: str = "-") -> list[str]:
     ignore_args = {"solver_dir", "instance", "cutoff_time", "seed", "objectives"}
     for key in args_dict:
         if key not in ignore_args and args_dict[key] is not None:
-            params.extend([prefix + str(key), str(args_dict[key])])
+            if postfix == " ":
```

Comment:
f"{prefix}{key}{postfix}{args_dict[key]}" instead? So no if else statement is needed due to the default value of postfix

Reply:
but if there is a space it will not work with subprocess.run